added add/remove to cart functionality

vidhi-thakur opened the pull request:
- added context for cart
- updated values in navbar, cart-management, and product page
- fetching products from api

check preview URL [here](https://deploy-preview-5--savvy-react-store.netlify.app/)

vidhi-thakur committed:
updated logo

vidhi-thakur committed:
configured axios

vidhi-thakur committed:
created a context for cart & fetching product data from API

vidhi-thakur committed:
updated context value in navbar and total amount in cart management page

sonishreyas commented on src/reducers/cartReducer.js:
```diff
@@ -0,0 +1,18 @@
+    totalAmount: 0
+}
+
+function cartReducer(state, action) {
```
you can destructure action here into type and payload

sonishreyas reviewed (approved):
The code looks good and easy to understand. I had 1 suggestion which you can implement. Rest looks good

NeerajVodala commented on src/components/cart-management/cart-management-card/CartManagementCard.js:
```diff
@@ -1,11 +1,29 @@
 
-function CartManagementCard({ title, author, rating, description, price, cartImage, }) {
+function CartManagementCard({ id, title, author, rating, description, price, productImage, }) {
+    const [, dispatch] = useCart();
```
can use object here instead of array

NeerajVodala reviewed (approved):
app is breaking on refresh. Add redirects to public folder.
Rest everything looks good.

vidhi-thakur commented:
Thank you @NeerajVodala for review and reporting the issue, will definitely fix  the bug.
Thank you @sonishreyas  for your suggestion

will merge the code after making the changes

vidhi-thakur committed:
refractor

- added _redirect file
- added CTA for empty cart page
- destructured object (action) in cart reducer

vidhi-thakur committed:
moved files location
